roles/matrix-base/tasks/validate_config.yml

```diff
@@ -6,6 +6,6 @@
       The `{{ item }}` variable defined in your configuration is not used by this playbook anymore.
       User/group creation is now dynamic. You can remove these variables from your configuration, as they have no effect on anything.
   when: "item in vars"
-  with_items:
-    - 'matrix_user_uid'
-    - 'matrix_user_gid'
+  with_items: []
+          #- 'matrix_user_uid'
+          #- 'matrix_user_gid'
```

Comment:
In git is is a good practice to remove such code and later dig trough the history if you need it again.